src/client/components/MainHeader/MainHeader.tsx

```diff
@@ -32,56 +38,245 @@ function OtapLabel() {
   ) : null;
 }
 
+const AmsMegaMenuClassname = 'ams-mega-menu';
+const BurgerMenuToggleBtnId = 'BurgerMenuToggleBtn';
+
+function isTargetWithinMenu(target: any) {
+  const LinkContainer =
+    document.getElementsByClassName(AmsMegaMenuClassname)[0];
```

Comment:
Je kunt hier ook `document.querySelector('.ams-mega-menu')` gebruiken of een id geven en dan `document.getElementById()`.